src/pages/Profile.js

```diff
@@ -48,12 +64,48 @@ function Profile({
     )
   );
 
+  const slugify = word =>
+    word
+      .toString()
+      .trim()
+      .toLowerCase()
+      .replace(/\s+/g, '-')
+      .replace(/[^\w-]+/g, '')
+      .replace(/-+/g, '-')
+      .replace(/^-+/, '')
+      .replace(/-+$/, '');
```

Comment:
Would this be useful in util?

Reply:
I have moved the function to utils folder